_posts/2024-05-23-pyos-call-for-editors.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: single
+title: "Volunteers needed: pyOpenSci Editorial Board"
+excerpt: "pyOpenSci is looking for volunteers from a wide array of scientific backgrounds to join our Editorial Board."
+author: "Jesse Mostipak"
+permalink: /blog/pyos-call-for-editors-may-2024.html
+header:
+  overlay_image: images/blog/headers/23-05-2024-call-for-editors.png
+  overlay_filter: 0.6
+categories:
+  - blog-post
+  - community
+classes: wide
+toc: true
+comments: true
+---
+## Join the pyOpenSci Editorial Board
+Thanks to the efforts of our incredible team of volunteers, pyOpenSci is growing! As a result, we're currently seeing a large number of package submissions to [our open peer review process](https://www.pyopensci.org/about-peer-review/index.html). In order to continue supporting scientists in the development of open source scientific software, we're looking to add volunteers to our Editorial Board.
```

Comment:
```suggestion
Thanks to the efforts of our incredible team of volunteers, pyOpenSci is growing! As a result, we're currently seeing a large number of package submissions to [our open peer review process](https://www.pyopensci.org/about-peer-review/index.html). In order to continue supporting scientists in the development of open source scientific software, we're looking to add volunteer editors to our Editorial Board.
```

Reply:
minor but we also have a triage team that i consider a part of the editorial board ... it's a way to allow someone to step onto the board that might not be ready to be a full editor. it's how isabel started with us!